samples/contrib/aws-samples/README.md

```diff
@@ -0,0 +1,198 @@
+# Sample AWS SageMaker Kubeflow Pipelines 
+
+This folder contains many example pipelines which use [AWS SageMaker Components for KFP](https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker). Following sections explain the setup you need to run these pipelines. Once you are done with the setup, [simple_train_pipeline](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/aws-samples/simple_train_pipeline) is a good place to start if you have never used these components before.
+
+
+
+## Prerequisites 
+
+1. You need a cluster with Kubeflow installed on it. [Install Kubeflow on AWS cluster](https://www.kubeflow.org/docs/aws/deploy/install-kubeflow/)
+2. Install the following on your local machine or EC2 instance (These are recommended tools. Not all of these are required)
+    1. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html). If you are using an IAM user, configure your [Access Key ID, Secret Access Key](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) and preferred AWS Region by running:
+       `aws configure`  
+    2. [aws-iam-authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html) version 0.1.31 and above  
+    3. [eksctl](https://github.com/weaveworks/eksctl) version above 0.15  
+    4. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) version 1.17 and above  
+    5. [KFP SDK](https://www.kubeflow.org/docs/pipelines/sdk/install-sdk/#install-the-kubeflow-pipelines-sdk) (installs the dsl-compile and kfp cli)
+    6. [jq](https://stedolan.github.io/jq/download/)
+
+
+## IAM Permissions 
+
+To use AWS KFP Components the KFP pods need access to AWS SageMaker.
+There are two ways you can give it access to SageMaker. 
+
+**Option 1]** (Recommended) [IAM roles for service account](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) 
```

Comment:
This square bracket looks a little odd. I think just the bold illuminates it enough.

Reply:
We should stipulate beforehand that this method is requires the cluster be running on EKS. While OIDC can exist outside EKS, this documentation will not do that use-case justice.

Reply:
added a line

Reply:
> This square bracket looks a little odd. I think just the bold illuminates it enough.

Square brackets are for hyperlink

Reply:
I meant square brackets on the `Option` text.